src/wp-includes/class-wp-block-patterns-registry.php

```diff
@@ -152,6 +152,25 @@ public function unregister( $pattern_name ) {
 		return true;
 	}
 
+	/**
+	 * Prepares the content of a block pattern.
+	 *
+	 * @param array $pattern     Registered pattern properties.
+	 * @param array $block_hooks The list of hooked blocks.
+	 * @return string The content of the block pattern.
+	 */
+	private function prepare_content( $pattern, $block_hooks ) {
```

Comment:
Good idea to extract this 👍

Reply:
```suggestion
	private function prepare_content( $pattern, $hooked_blocks ) {
```

🙈

Reply:
Addressed with https://github.com/WordPress/wordpress-develop/pull/5399/commits/21efb9edb6f4ef1881cdb6bed8fa1c97c65ba0f6.